pandas/core/panel.py

```diff
@@ -877,7 +878,7 @@ def to_frame(self, filter_observations=True):
         """
         _, N, K = self.shape
 
-        if filter_observations:
```

Comment:
Tested it: it's already being handled correctly by the decorator:
Both, filter_observations and dropna map correctly to the desired behavior.
Hence IMO I don't need to handle it.